### PR TITLE
+libxslt

### PR DESCRIPTION
**Pull request by mfts**
chatgpt said it was a requirement for postgres, so i built it. turns out it's not a requirement. but the build process was easy (similar to libxml2)

### CODE REVIEW

**Comment by mfts** [comment]
"Insanity is doing the same thing over and over and expecting different results." 

will continue later on debugging linux test

**Comment by jhheider** [comment]
Nested headers are annoying. Some tools are resiliant, and some are not. See e.g. https://github.com/teaxyz/pantry.core/blob/adaaf3af9417051f02a211b1c3d25c5c0c1314e9/projects/python.org/package.yml#L107-L108. Although, often, simply adding `pkgconfig` as a build dep is sufficient.

File: projects/gnome.org/libxslt/package.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
+  script: |
+    ./configure $ARGS
+    make --jobs {{ hw.concurrency }}
+    make install
+  env:
+    ARGS:
+      - --prefix={{prefix}}
+      - --without-python  # mysterious build failure in GHA macOS-11
```
Comment (mxcl) [inline projects/gnome.org/libxslt/package.yml]: the comment is copy pasted from libxml2. Does it apply? Or did you just YOLO paste?

Comment (mfts) [reply projects/gnome.org/libxslt/package.yml]: Applies here as well. Without it it uses Python and crashes the build

Comment (mfts) [reply projects/gnome.org/libxslt/package.yml]: But yes I checked the `libxml2` build first because the docs say the build instructions are very similar.

Comment (mxcl) [reply projects/gnome.org/libxslt/package.yml]: k good stuff ☺️

**Comment by mfts** [comment]
@mxcl how is `xslt-config.patch.in` used? Is that file automatically called by `xslt-config`?

**Comment by mxcl** [comment]
@mfts you can see the patch being applied on the first line of the build script

**Comment by mfts** [comment]
@mxcl I only looked at the `test`. Learned something new 👍

**Pull request merged**